aws_adfs/roles_assertion_extractor.py

```diff
@@ -37,6 +37,21 @@ def extract(html):
     # Note the format of the attribute value is provider_arn, role_arn
     principal_roles = [role for role in aws_roles if ':saml-provider/' in role[0]]
 
+    # File contains a list of AWS account id's and names
+    # map_file = requests.get('https://s3.amazonaws.com/eis-aws-accounts/account_map.json')
+    map_file = requests.get('https://aafn.ebsco.cloud')
```

Comment:
I think that's your api gateway supported by the lambda function - I think that there is a flow in the design. First you will be charged for every call (please note, that this tool can be used everywhere around the world), besides that I don't see the need to use lambda to fetch account names. Friendly account names are provided by ADFS server and `aws-adfs` already use them to group roles.